assets/prototype/domain/shared/predicates/datetimes/filters/tests/data.ts

```diff
@@ -0,0 +1,56 @@
+import { Datetime } from '../../../../../eventEditor/data/types';
+
+const dates: Datetime[] = [
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: -1,
+		isActive: true,
+		sold: 0,
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: true,
+		sold: 5,
+		status: 'ACTIVE',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: true,
+		sold: 6,
+		status: 'ACTIVE',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: true,
+		sold: 7,
+		status: 'ACTIVE',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: true,
+		sold: 8,
+		status: 'ACTIVE',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: false,
+		isUpcoming: true,
+		sold: 9,
+		status: 'ACTIVE',
+	},
+	{
+		id: 'RGF0ZXRpbWU6ODM=',
+		capacity: 10,
+		isActive: false,
+		isUpcoming: true,
+		sold: 10,
+		status: 'ACTIVE',
+	},
+];
```

Comment:
This should ideally be in `nodes` in `/domain/eventEditor/data/queries/datetimes/test/data.ts`

Reply:
Reused test data from that source in some places, in other it was throwing some errors so kept it simple.

Reply:
Alright 👍